test/unit/htmlSanitizer.spec.js

```diff
@@ -41,5 +41,13 @@ describe('htmlSanitizer', function() {
       expect($circle.attr('stroke-width')).toBeTruthy();
       expect($circle.attr('fill')).toBeTruthy();
     });
+
+    it('Remove attributes with invalid value', function() {
+      expect(htmlSanitizer('<a href="javascript:alert();">xss</a>', true)).toEqual('<a>xss</a>');
```

Comment:
If you want to compare the value of primitive type, we recommend using `toBe`.

Reply:
https://github.com/nhn/tui.editor/blob/7407f58dd979df066b20144b7f4925e514a2b272/test/unit/htmlSanitizer.spec.js#L12

Do you want me to fix this also?

Reply:
@LunaTK Yes, you can fix it together.